templating/templating.go

```diff
@@ -43,6 +45,214 @@ type trigger struct {
 	Name string `json:"name"`
 }
 
+func FilterKeys(source template.FuncMap, keys []string) template.FuncMap {
+	result := template.FuncMap{}
+	for _, key := range keys {
+		value, found := source[key]
+		if found {
```

Comment:
```suggestion
		if value, ok := source[key], ok {
```

Reply:
Does not work for me
![image](https://github.com/moira-alert/moira/assets/55148949/f5a805ca-936e-434e-84f7-0099946b6ca2)

Reply:
if value, ok := source[key], ok { => if value, ok := source[key]**;** ok {

Reply:
not ,
use ; in this :)

Reply:
Sorry, it was a typo

Reply:
fixed